addons/l10n_ug/data/account_tax_report_data.xml

```diff
@@ -0,0 +1,579 @@
+                                <field name="label">allowed</field>
+                                <field name="engine">external</field>
+                                <field name="formula">most_recent</field>
+                                <field name="subformula">editable</field>
+                                <field name="figure_type">boolean</field>
+                            </record>
+                        </field>
+                    </record>
+                    <record id="tax_report_ug_line_F29" model="account.report.line">
+                        <field name="name">29. Input tax directly attributable to Taxable Sales</field>
+                        <field name="code">UG_TAX_F29</field>
+                        <field name="expression_ids">
+                            <record id="tax_report_ug_line_F29_tax_tag" model="account.report.expression">
+                                <field name="label">allowed</field>
+                                <field name="engine">external</field>
+                                <field name="formula">most_recent</field>
+                                <field name="subformula">editable;rounding=2</field>
+                            </record>
+                        </field>
+                    </record>
+                    <record id="tax_report_ug_line_F30" model="account.report.line">
+                        <field name="name">30. Input tax directly attributable to Exempt Sales (Disallowed) and non-creditable input tax</field>
+                        <field name="code">UG_TAX_F30</field>
+                        <field name="expression_ids">
+                            <record id="tax_report_ug_line_F30_base_tag" model="account.report.expression">
+                                <field name="label">disallowed</field>
+                                <field name="engine">external</field>
+                                <field name="formula">most_recent</field>
+                                <field name="subformula">editable;rounding=2</field>
+                            </record>
+                        </field>
+                    </record>
+                    <record id="tax_report_ug_line_F31" model="account.report.line">
+                        <field name="name">31.Input tax apportioned to Taxable Sales using selected formula</field>
+                        <field name="code">UG_TAX_F31</field>
+                        <field name="expression_ids">
+                            <record id="tax_report_ug_line_F31_tax_tag" model="account.report.expression">
+                                <field name="label">allowed</field>
+                                <field name="engine">aggregation</field>
+                                <field name="formula">(1 - UG_TAX_FM.allowed) * UG_TAX_F31.hidden_computation_of_F31_using_normal_formula + UG_TAX_FM.allowed * UG_TAX_F31.hidden_computation_of_F31_using_sam_formula</field>
```

Comment:
We can break this line, it's a bit long :smile: (happens several time on the file)

Reply:
![that's what she said](https://media.tenor.com/Jlym3w00uR4AAAAd/michael-scott-thats-what-she-said.gif)